src/analyzer/analyzerkey.h

```diff
@@ -6,18 +6,18 @@
 
 #include "analyzer/analyzer.h"
 #include "analyzer/plugins/analyzerplugin.h"
+#include "analyzer/plugins/analyzerpluginsupport.h"
 #include "preferences/keydetectionsettings.h"
 #include "preferences/usersettings.h"
 #include "track/track_decl.h"
 #include "util/memory.h"
 
-class AnalyzerKey : public Analyzer {
+class AnalyzerKey : protected AnalyzerPluginSupportInfo, public Analyzer {
```

Comment:
protected inheritance is uncommon in C++ because in addition to the already protected functions it makes the public functions also protected. It looks like in this case you can inherit privately. Otherwise inherit public and make all functions protected.